### PR TITLE
Fix broken directory structure of Imagenet format

### PR DESCRIPTION
**Pull request by cih9088**
### Summary
Fixed messed up broken directory structure.

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [ ] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: docs/source/docs/explanation/formats/imagenet.md [hunk 1]
```diff
@@ -39,38 +39,41 @@ Load ImageNet dataset through the Python API:
 import datumaro as dm
 
 dataset = dm.Dataset.import_from('<path_to_dataset>', format='imagenet_txt')
+# or
+dataset = dm.Dataset.import_from('<path_to_dataset>', format='imagent')
```
Comment (JihwanEom) [inline docs/source/docs/explanation/formats/imagenet.md]:
```suggestion
dataset = dm.Dataset.import_from('<path_to_dataset>', format='imagenet')
```

File: docs/source/docs/explanation/formats/imagenet.md [hunk 2]
```diff
@@ -39,38 +39,41 @@ Load ImageNet dataset through the Python API:
...
+  ├── label_0
+  │   ├── <image_name_0>.jpg
+  │   ├── <image_name_1>.jpg
+  │   ├── <image_name_2>.jpg
+  │   ├── ...
+  ├── label_1
+  │    ├── <image_name_0>.jpg
+  │    ├── <image_name_1>.jpg
+  │    ├── <image_name_2>.jpg
+  │    ├── ...
```
Comment (vinnamkim) [inline docs/source/docs/explanation/formats/imagenet.md]: These lines seem indented by 1 space compared to others.

File: docs/source/docs/explanation/formats/imagenet.md (hunk 1 above)
Comment (cih9088) [reply docs/source/docs/explanation/formats/imagenet.md]: Fixed in https://github.com/openvinotoolkit/datumaro/pull/985/commits/8e5ebb817a14b8e73fac486a1171bbac5e8dc740

File: docs/source/docs/explanation/formats/imagenet.md (hunk 2 above)
Comment (cih9088) [reply docs/source/docs/explanation/formats/imagenet.md]: Fixed in https://github.com/openvinotoolkit/datumaro/pull/985/commits/8e5ebb817a14b8e73fac486a1171bbac5e8dc740

**Review by vinnamkim — APPROVED** [review]
LGTM.

**Review by JihwanEom — APPROVED** [review]

**Pull request merged**